DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -82,9 +82,10 @@ GROUP BY
                     WHERE
                         CustomisationTutorials.CustomisationID = @customisationId
                         AND Sections.SectionID = @sectionId
-                        AND (Sections.ArchivedDate IS NULL)
+                        AND Sections.ArchivedDate IS NULL
```

Comment:
What about the id for the next section, are we checking that isn't archived?

Reply:
This line is just some brackets getting removed for consistency I think. The query isn't getting the next section ID here, because I haven't merged in Shiv's latest changes. I'll add those in soon 🙂

Reply:
Added in #157 🙂